yarn.lock

```diff
@@ -12377,11 +12402,16 @@ ws@8.11.0, ws@~8.11.0:
   resolved "https://registry.yarnpkg.com/ws/-/ws-8.11.0.tgz#6a0d36b8edfd9f96d8b25683db2f8d7de6e8e143"
   integrity sha512-HPG3wQd9sNQoT9xHyNCXoDUa+Xw/VevmY9FoHyQ+g+rrMn4j6FB4np7Z0OhdTgjx6MgQLK7jwSy1YecU1+4Asg==
 
-ws@^8.13.0, ws@^8.14.1:
+ws@^8.13.0:
   version "8.16.0"
   resolved "https://registry.yarnpkg.com/ws/-/ws-8.16.0.tgz#d1cd774f36fbc07165066a60e40323eab6446fd4"
   integrity sha512-HS0c//TP7Ina87TfiPUz1rQzMhHrl/SG2guqRcTOIUYD2q8uhUdNHZYJUaQ8aTGPzCh+c6oawMKW35nFl1dxyQ==
```

Comment:
Question: Is the dependency to `8.16.0` a runtime dependency? As far as I can tell, it is pulled from `jsdom`. We should probably update that as well.

Reply:
@msujew, thanks for having a look to this PR! 
Yes, the `^8.13.0` dependency comes from jsdom 21.1.1. The latest version of [jsdom 24.1.0](https://nuget.qite.be/feeds/NPM/jsdom/24.1.0/dependencies)) depends on ws as  `^8.17.0`. 


Shall I update jdom to this latest version? 
I could also change ws reexported dependency from package/core/package.json to `^8.17.1` instead of `^8.18.0`. Not sure what has been the best practice in the project in the past. `8.18.0` is the latest available, `8.17.1` is the first being know without DoS vulnerabiltity.